### PR TITLE
Bug 1737558: Adding PVC's to Namespace resource collection

### PR DESCRIPTION
**Pull request by sferich888**
@juanluisvaladas can you please review this. 
cc: @deads2k

### CODE REVIEW

**Comment by deads2k** [comment]
@soltysh @sanchezl this looks reasonable to me, but it's another reason that having two sources of truth is a problem.

File: pkg/cmd/inspect/namespace.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ func namespaceResourcesToCollect() []schema.GroupResource {
 		{Resource: "events"},
 		{Resource: "configmaps"},
 		{Resource: "secrets"},
+		{Resource: "pvcs"},
```
Comment (deads2k) [inline pkg/cmd/inspect/namespace.go]: fully specify, no shortnames

Comment (sferich888) [reply pkg/cmd/inspect/namespace.go]: Note: s/pvcs/PersistentVolumeClaims

**Comment by juanluisvaladas** [comment]
lgtm

**Comment by sferich888** [comment]
@juanluisvaladas @deads2k updated to use a full name.

**Comment by sferich888** [comment]
/bugzilla refresh

@deads2k can I get a /lgtm?

**Comment by deads2k** [comment]
/approve

@sanchezl @soltysh be sure you don't drop this.

**Pull request merged**